WeatherService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Weather_Monitoring_System.Bots;
+using Weather_Monitoring_System.DataParsers;
+using Weather_Monitoring_System.Models;
+
+namespace Weather_Monitoring_System
+{
+    public class WeatherService
+    {
+        private readonly IWeatherDataParser _dataParser;
+
+        private readonly List<Bot> _botServices = [new RainBotService(), new SnowBotService(), new SunBotService()];
+
+        public WeatherService(IWeatherDataParser dataParser)
+        {
+            _dataParser = dataParser;
+        }
+
+        public WeatherData ParseToWeatherData(string data)
+        {
+            return _dataParser.TryParse(data);
+        }
+
+        public void AvtivatBots(WeatherData weatherData)
```

Comment:
This method's name doesn't convey the intention of the method.
The intention of the name would suggest that this method will activate a bot. However, what it does is to check if the bot is active!